src/robot.impl.cc

```diff
@@ -456,6 +456,22 @@ namespace hpp {
         }
       }
 
+      void Robot::setHandlePositionInJoint (const char* handleName,
+          const ::hpp::Transform_ position)
+      {
+	try {
+          DevicePtr_t robot = getRobotOrThrow (problemSolver());
+          HandlePtr_t handle = robot->handles.get (handleName);
+          if (!handle)
+            throw Error ("This handle does not exists.");
```

Comment:
Can you change the error message into "robot (robot name) does not have any handle named (handle name)" ?